FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,37 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .success(let (data, httpResponse)):
+				completion(FeedImageMapper.map(data: data, from: httpResponse))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+final class FeedImageMapper {
+	private struct RemoteObj: Decodable {
+		var items: [RemoteFeedImage]
```

Comment:
`items` are never changed thus it can be `let` instead of `var`
```suggestion
		let items: [RemoteFeedImage]
```